Create folder design with all mockups (#54)
KrispyTech/airneis: docs/design/mockups/web_mobile/layout.md

# Layout Mockup

![Layout Mockup](./images/layout.png)

This is the main layout that will be used for all other pages.
We can see the header, which contains "ÀIRNEIS".

> - The search icon allows we to search for all the products on the site or on mobile.

Reviewer: ```suggestion
- The search icon allows us to search for all the products on the website or on the mobile app.
```
